crates/bitwarden-wasm-internal/src/client.rs

```diff
@@ -21,7 +21,7 @@ pub struct BitwardenClient(pub(crate) Client);
 impl BitwardenClient {
     #[allow(missing_docs)]
     #[wasm_bindgen(constructor)]
-    pub fn new(settings: Option<ClientSettings>, token_provider: JsTokenProvider) -> Self {
+    pub fn new(token_provider: JsTokenProvider, settings: Option<ClientSettings>) -> Self {
```

Comment:
Question: Why the argument order change?

Reply:
Because token_provider is required and settings is optional. Having them the other way be clunky in js.